docs/cli/auth/admin.md

```diff
@@ -104,3 +108,162 @@ function App() {
 
 export default withAuthenticator(App, true);
 ```
+
+</amplify-block>
+
+<amplify-block name="iOS">
+
+1. Initialize Amplify API. Refer to [Getting Started with Amplify.API for REST](~/lib/restapi/getting-started.md) for more details. 
+
+You should have the initialization code including the imports:
+
+```swift
+import Amplify
+// If you are using Swift Package Manager
+import AWSAPIPlugin
+// or if you are using Cocoapods
+import AmplifyPlugins
+```
+
+and code that adds `AWSCognitoAuthPlugin`, `AWSAPIPlugin`, and configures it.
+
+```swift
+try Amplify.add(plugin: AWSCognitoAuthPlugin())
+try Amplify.add(plugin: AWSAPIPlugin())
+try Amplify.configure()
+```
+
+2. Sign in using `Amplify.Auth`. See [Amplify.Auth](~/lib/auth/getting-started.md) to learn more about signing up and signing in a user.
+
+3. Use the following in your app to add a user to the Group.
+
+```swift
+func addToGroup(username: String, groupName: String) {
+    let path = "/addUserToGroup"
+    let body = "{\"username\":\"\(username)\",\"groupname\":\"\(groupName)\"}".data(using: .utf8)
+    let request = RESTRequest(path: path, body: body)
+    Amplify.API.post(request: request) { result in
+        switch result {
+        case .success(let data):
+            print("Response Body: \(String(decoding: data, as: UTF8.self))")
+        case .failure(let error):
+            if case let .httpStatusError(statusCode, response) = error,
+                let awsResponse = response as? AWSHTTPURLResponse,
+                let responseBody = awsResponse.body
+            {
+                print("StatusCode: \(statusCode) Response Body: \(String(decoding: responseBody, as: UTF8.self))")
+            }
+        }
+    }
+}
+
+addToGroup(username: "richard", groupName:  "Editors")
+```
+
+4. Use the following to list the users in the Group.
+
+```swift
+func listEditors(groupName: String, limit: Int, nextToken: String? = nil) {
+    let path = "/listUsersInGroup"
+    var query = ["groupname": groupName,
+                  "limit": String(limit)]
+    if let nextToken = nextToken {
```

Comment:
nit: I'm not not a swift guru, is this a good practice to reuse the same name as the parameter?

Reply:
I think this is fine because we need to unwrap the parameter `nextToken` which is optional and the type completion in IDE controls what can be accessed from the variable at line 169 and line 171. Similarly, in https://docs.swift.org/swift-book/LanguageGuide/OptionalChaining.html swift docs also shows a few examples of doing the same
```swift
if let buildingNumber = buildingNumber, let street = street {
```

Reply:
Thanks for the explanation :)